Games/core/Meeting.js

```diff
@@ -638,7 +638,8 @@ module.exports = class Meeting {
                     (!this.multi && this.votes[member.id] == null) ||
                     (this.multi && selections.length < this.multiMin && selections.indexOf("*") == -1)
                 ) {
-                    this.game.vegPlayer(member.player);
+                    const playerIsKicklynched = this.actionName === "Village Vote" && this.finalTarget === member.id;

```

Comment:
I feel like the game doesn't need to veg the player, just check IsKickEliminated --> then pass